packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,46 @@
 
 ## Changelog
 
+### Aug 15, 2023
+
+New features:
+
+- A new app template `AI Chat Bot` to help you get started with building a GPT-like chat bot with AI capabilities using `Teams AI Library`.
+  ![AI Bot](https://github.com/OfficeDev/TeamsFx/assets/11220663/86a90d2a-efc3-4d8b-9e8c-5d34a1e8c081)
+- Onboarded a new sample `One Productivity Hub using Graph Toolkit with SPFx` that shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit components and SharePoint provider.
+    ![SPFx Sample](https://github.com/OfficeDev/TeamsFx/assets/11220663/084ac508-49ea-4b30-854c-8b4d578ff6ee)
+- Added codelens to `teamsapp.yml` file to help you execute life-cycle commands easily after editing the file.
```

Comment:
Added CodeLens hints to the `teamsapp.yml` file to help you run life-cycle commands easily after editing the file.

Reply:
Updated